backend/SyncReverseProxy/Services/ProxyEventsService.cs

```diff
@@ -12,13 +12,20 @@ public ProxyEventsService(ILexProxyService lexProxyService)
         _lexProxyService = lexProxyService;
     }
 
-    public ValueTask OnResumableRequest(HttpContext context)
+    public async Task OnResumableRequest(HttpContext context)
     {
-        //todo update project last change
-        return ValueTask.CompletedTask;
+        if (context.Request.Path.StartsWithSegments("/api/v03/finishPushBundle"))
```

Comment:
I don't think there is a `finishPushBundle` 🤔.
[honeycomb query](https://ui.honeycomb.io/sil-language-forge/environments/prod?query=%7B%22time_range%22%3A86400%2C%22granularity%22%3A0%2C%22breakdowns%22%3A%5B%22http.url%22%5D%2C%22calculations%22%3A%5B%7B%22op%22%3A%22COUNT%22%7D%5D%2C%22filters%22%3A%5B%7B%22column%22%3A%22http.url%22%2C%22op%22%3A%22contains%22%2C%22value%22%3A%22api%2Fv03%22%7D%2C%7B%22column%22%3A%22http.url%22%2C%22op%22%3A%22contains%22%2C%22value%22%3A%22push%22%7D%2C%7B%22column%22%3A%22http.url%22%2C%22op%22%3A%22contains%22%2C%22value%22%3A%22finish%22%7D%5D%2C%22filter_combination%22%3A%22AND%22%2C%22orders%22%3A%5B%7B%22op%22%3A%22COUNT%22%2C%22order%22%3A%22descending%22%7D%5D%2C%22havings%22%3A%5B%5D%2C%22limit%22%3A100%7D).

Looks like it was [commented out](https://github.com/sillsdev/chorus/blob/74871cb48a472282142e60e19088702825af2692/src/LibChorus/VcsDrivers/Mercurial/HgResumeTransport.cs#L436C46-L436C46) 😕. There is one other [call to it](https://github.com/sillsdev/chorus/blob/74871cb48a472282142e60e19088702825af2692/src/LibChorus/VcsDrivers/Mercurial/HgResumeTransport.cs#L416C30-L416C30), but only when the S/R fails.

So, I think we need to copy [this check](https://github.com/sillsdev/chorus/blob/74871cb48a472282142e60e19088702825af2692/src/LibChorus/VcsDrivers/Mercurial/HgResumeTransport.cs#L423).

Something like:
```
if (!error.HasValue &&
	(response.Status == PushStatus.Complete || (req.Offset + req.ChunkSize) >= req.BundleSize))
```

Reply:
there is, https://ui.honeycomb.io/sil-language-forge/environments/prod/result/i56Pk5sDRgR